### PR TITLE
Add changelogs for 10.0.2, 9.0.6 and 8.0.13

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,37 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 10.0.2 – 2020-11-17
+### Fixed
+- Reduce performance impact caused by the existance of the emoji picker
```
Comment (PVince81) [inline CHANGELOG.md:6]: existence

(but not this one https://www.imdb.com/title/tt0120907/)

**Review by PVince81 — APPROVED** [review]
Fix the typo then :+1:

**Comment by PVince81** [comment]
and add https://github.com/nextcloud/spreed/pull/4620 separately I guess ?

**Comment by nickvergessen** [comment]
> and add #4620 separately I guess ?

I don't think it matters too much for users

**Pull request merged**